android/src/play/java/com/tailscale/ipn/Google.java

```diff
@@ -28,7 +28,7 @@ static void googleSignIn(Activity act, String serverOAuthID, int reqCode) {
 					.build();
 				GoogleSignInClient client = GoogleSignIn.getClient(act, gso);
 				Intent signInIntent = client.getSignInIntent();
-				App.startActivityForResult(act, signInIntent, reqCode);
+				App.Companion.startActivityForResult(act, signInIntent, reqCode);
```

Comment:
If you don't want to have to use Companion here, you could annotate `startActivityForResult` with `@JVMStatic`.